### PR TITLE
Add Copy Button

### PR DESCRIPTION
**Pull request by meowmeowmeowcat**
- This allows folks to copy commands directly with buttons.

### CODE REVIEW

File: source/conf.py [hunk 1]
```diff
@@ -35,8 +35,13 @@
     'sphinx.ext.intersphinx',
     'sphinx.ext.todo',
     'sphinx_inline_tabs',
+    'sphinx_copybutton',
 ]
 
+# config for copy button
+copybutton_prompt_text = r">>> |\.\.\. "
```
[the remark this answers is not in the document]
Comment (meowmeowmeowcat) [reply source/conf.py]: Oh, I see. However, we won't remove prompts like `$` in the future?

File: source/_static/overrides.css [hunk 2]
```diff
@@ -0,0 +1,8 @@
+/* Overrides for theme and plugin styles */
+
+/* https://github.com/executablebooks/sphinx-copybutton/blob/v0.4.0/sphinx_copybutton/_static/copybutton.css */
+
+button.copybtn img {
+    /* Fix alignment for pypa_theme */
+    padding: 0;
+}
```
Comment (bhrutledge) [inline source/_static/overrides.css:8]: The icon alignment was bugging me, so I took the liberty to fix it. 😄  This (and the associated  `html_` changes in `conf.py`) seemed to follow the suggestion at https://sphinx-copybutton.readthedocs.io/en/latest/#customize-the-css.

I'd love other folks' thoughts on this.

**Review by bhrutledge — APPROVED** [review]
Very cool, thank you!

File: source/conf.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (bhrutledge) [reply source/conf.py]: I just opened https://github.com/pypa/packaging.python.org/issues/947 for that, but this covers the existing use, plus any prompts that remain afterwards.

File: source/_static/overrides.css (hunk 2 above)
Comment (pradyunsg) [reply source/_static/overrides.css:8]: I'd prefer we get this fixed upstream.

Comment (pradyunsg) [reply source/_static/overrides.css:8]: That said, this is not a blocking concern -- rather I'm saying that this is an issue that we should follow up upstream for.

**Review by webknjaz — APPROVED** [review]

File: source/_static/overrides.css (hunk 2 above)
Comment (bhrutledge) [reply source/_static/overrides.css:8]: Thanks. By upstream, do you mean `pypa_theme`, or sphinx-copybutton? I'm guessing the alignment issue is specific to `pypa_theme`; the button renders fine in the [sphinx-copybutton docs](https://sphinx-copybutton.readthedocs.io/en/latest/), which also says it's designed to work with Alabaster.

**Pull request merged**